manager_test.go

```diff
@@ -3,13 +3,55 @@ package taskmanager_test
 import (
 	"context"
 	"errors"
+	"log"
 	"testing"
 	"time"
 
-	logging "github.com/ipfs/go-log/v2"
 	"github.com/plexsysio/taskmanager"
 )
 
+type mockLogging struct{}
+
+func (m mockLogging) Tracef(format string, args ...interface{}) {
+	log.Printf(format, args...)
+}
+
+func (m mockLogging) Trace(args ...interface{}) {
+	log.Println(args...)
+}
+
+func (m mockLogging) Debugf(format string, args ...interface{}) {
+	log.Printf(format, args...)
+}
+
+func (m mockLogging) Debug(args ...interface{}) {
+	log.Println(args...)
+}
+
+func (m mockLogging) Infof(format string, args ...interface{}) {
```

Comment:
Same comment. Ignore type variable in signature if you dont use it.

```suggestion
func (mockLogging) Infof(format string, args ...interface{}) {
```